istio.VERSION

```diff
@@ -1,13 +1,13 @@
 # DO NOT EDIT THIS FILE MANUALLY instead use
```

Comment:
revert this (maybe we can put it in gitignore or some way to not change the one from source tree when we use it)